docs/duration.md

```diff
@@ -386,25 +386,28 @@ d = Temporal.Duration.from('-PT8H30M');
 d.abs(); // PT8H30M
 ```
 
-### duration.**round**(_options_: object) : Temporal.Duration
+### duration.**round**(_roundTo_: object) : Temporal.Duration
 
 **Parameters:**
 
-- `options` (object): An object with properties representing options for the operation.
-  The following options are recognized:
-  - `largestUnit` (string): The largest unit of time to allow in the resulting `Temporal.Duration` object.
-    Valid values are `'auto'`, `'year'`, `'month'`, `'week'`, `'day'`, `'hour'`, `'minute'`, `'second'`, `'millisecond'`, `'microsecond'`, and `'nanosecond'`.
-    The default is `'auto'`.
-  - `smallestUnit` (string): The smallest unit of time to round to in the resulting `Temporal.Duration` object.
-    Valid values are `'year'`, `'month'`, `'week'`, `'day'`, `'hour'`, `'minute'`, `'second'`, `'millisecond'`, `'microsecond'`, and `'nanosecond'`.
-    The default is `'nanosecond'`, i.e. no rounding.
-  - `roundingIncrement` (number): The granularity to round to, of the unit given by `smallestUnit`.
-    The default is 1.
-  - `roundingMode` (string): How to handle the remainder, if rounding.
-    Valid values are `'halfExpand'`, `'ceil'`, `'trunc'`, and `'floor'`.
-    The default is `'halfExpand'`.
-  - `relativeTo` (`Temporal.PlainDateTime`): The starting point to use when converting between years, months, weeks, and days.
-    It must be a `Temporal.PlainDateTime`, or a value that can be passed to `Temporal.PlainDateTime.from()`.
+- `roundTo` (string | object): A required string or object to control the operation.
```

Comment:
I liked Yulia's terminology of "a configuration object" — maybe we can use that throughout? At least in the docs, TypeScript types, and tests. (I'm not sure if it's worth changing it in the spec text, since we already have GetOptionsObject there which shares terminology with ECMA-402.)

Reply:
Configuration is something that applies to a stateful thing; you don't really configure a single one-off method call :-/

Either way, I'd say that verbage would only apply to Duration round, since the others really are just options bags.

Reply:
@ptomato - are you referring to the parameter name `roundTo` or to the descriptive text _required string or object to control the operation` where I carefully avoided giving this thing a general name like "options" or "configuration object", specifically to avoid arguments like the one happening above?  😄